# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1304309395 -43200
# Node ID 845424adaf310cadba6360bdde963b9a6a19eb4a
# Parent  75a0da5d513ca9cf2944c1ec3c5acfdfe0c00bfa
trac #10764: fix doctest failures generated in gen_interpreters.py

diff --git a/sage/ext/gen_interpreters.py b/sage/ext/gen_interpreters.py
--- a/sage/ext/gen_interpreters.py
+++ b/sage/ext/gen_interpreters.py
@@ -3662,7 +3662,7 @@
         elided below):
             sage: print rdf_pxd
             # Automatically generated by ext/gen_interpreters.py.  Do not edit!
-            from python_object cimport PyObject
+            from cpython cimport PyObject
             from sage.ext.fast_callable cimport Wrapper
             ...
             sage: print rr_pxd
